manifests/fetch_artifact.pp

```diff
@@ -58,38 +58,34 @@
   $version,
   $install_path,
   $format,
-  $path        = undef,
+  $path        = '',
```

Comment:
Puppet 4 is unhappy with assigning strings to an empty value - why the change on these?